### ISSUE TITLE
Add variation count to experiment directory

### ISSUE THREAD

**Issue by aaronyan** [Automattic/abacus#747]
What do y'all think about adding the # of assignment groups / variations to the experiment directory? Seeing 3+ variations for an experiment will be a natural way to advertise A/B/n capabilities!

**Comment by jessie-ross** [comment]
Hmm, I'm not sure how useful it will be to have it there 🤔 

If we want to advertise A/B/n we could highlight it in the experiment creation wizard.

**Comment by jessie-ross** [comment]
It is very easy to add in, however.

**Comment by aaronyan** [comment]
Yeah, the usefulness of adding variation count to experiment directory is definitely on the low side. I was trying to find an A/B/n experiment during the meetup, and I was surprised how hard it was to find. This is unlikely a popular experience.
